frontend/src/components/Header/components/AuthButtons/index.tsx

```diff
@@ -47,14 +49,20 @@ function LoggedInButtons({ name, email }: { name?: string; email?: string }) {
   );
 
   function Content() {
+    const curatorAPIFeature = get(FEATURES.CURATOR_API);
     return (
       <Menu>
-        <MenuItem data-testid="user-email" text={`Logged in as: ${email}`} />
```

Comment:
is this MenuItem removed intentionally? Thanks!

Reply:
yes! Just matching the figma

Reply:
done!

Reply:
Appreciate it!